src/controllers/quest.js

```diff
@@ -343,6 +348,16 @@ class Quest extends Controller {
 						conditionString = conditionString.concat(cond)
 						break
 					}
+					case 37: {
+						const pokemons = []
+						condition.info.raid_pokemon_evolutions.forEach((pokemonId) => {
+							pokemons.push(monsterData[pokemonId].name)
+						})
+						const template = this.qdts.questConditions['37']
```

Comment:
This condition isn't defined anywhere and breaks current systems.

Reply:
Can you paste the error you're getting? What you say doesn't make sense to me because `case 37` is in case the parser sends a quest with the condition of 37. The line `const template = this.qdts.questConditions['37']` looks in your questdts.json file for the string template for forming the output text. I did forget to commit the example update so I pushed that but I would like to see your error since you can start the bot without that dts.

Reply:
There is an error when trying to access qdts.questConditions['37'] as the property doesn't exits. Thanks for updating the example json file.

Reply:
Let me know if updating the questdts solved that issue. If not, give me more info on what you were doing because poraclejs is running with this code. I haven't scanned a quest with condition 37 yet (I don't think it's been released)